### DISCUSSION TITLE
Android splash image

### DISCUSSION THREAD

**Discussion by Jzhenli** [beeware/briefcase#1969] [Q&A] [answered]
Hi, when open the android apk, it will display white page with icon for several seconds, how to set a background image for it? 
BTW, my tablet OS is android 10.

**Comment by freakboy3742** [comment]
This is Android's splash screen; the API that Toga exposes doesn't provide a mechanism for a full-screen image, primarily because you can't control the size of the device you'll be displaying the app on.

You *can* control the background color, though - so it doesn't need to be white. Use the [`splash_background_color`](https://briefcase.readthedocs.io/en/stable/reference/configuration.html#splash-background-color) configuration item to control the splash background color.

**Reply by Jzhenli** [reply]
@freakboy3742  thanks for your reply. what the optional background color can be set?  I have search in the doc, and haven't find it.

**Reply by freakboy3742** [reply, marked as answer]
I provided a link to the documentation, which literally answers this question in the first sentence: A hexadecimal RGB color value (e.g., #6495ED).
